server.js

```diff
@@ -13,6 +13,7 @@ const server = express()
 server.use(function(req, res, next) {
     res.header("Access-Control-Allow-Origin", "http://localhost:4200"); // update to match the domain you will make the request from
     res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    res.header('Access-Control-Allow-Methods', 'GET, PATCH, PUT, POST, DELETE, OPTIONS');
```

Comment:
**note:** Coors was upset about DELET so we need to be sure to "allow" the methods we want the `localhost:4200` to use.